docs/releases.rst

```diff
@@ -36,6 +27,42 @@ contrast to the :ref:`/spec/core/contact` key, this field is not
 supposed to be updated and can be useful when trying to track down
 the original author for consultation.
 
+Add support for running playbooks from Ansible collections specified
+using the ``namespace.collection.playbook`` notation.
+
+Added ``--dry`` option for the ``beaker`` provision plugin. When
+used it prints the Beaker Job XML without submitting it.
+
+`Results specification documentation`__ has now a dedicated place
+in the specification for improved discoverability.
+
+__ https://tmt.readthedocs.io/en/latest/spec/results.html
+
+The ``rpm-ostree`` package installation now includes the
+``--assumeyes`` option for improved compatibility.
+
+Verbosity levels in ``tmt * show`` commands are now honored.
+
+Large local variables in ``log.txt`` are now shortened. Added
+verbosity levels: locals (shortens large locals) and
+full (logs everything).
+
+Fixed an issue where ``execute`` step incorrectly attempted to run
+disabled ``discover`` phases.
+
+Pre-defined order values of `special phases`__ were documented.
```

Comment:
Same as above, no absolute URL, `:ref:` is the right one:

```
:ref:`/spec/plans/prepare`
```

Reply:
thanks, changed.